docs/man/docker-create.1.md

```diff
@@ -102,7 +102,7 @@ IMAGE [COMMAND] [ARG...]
                                'host': use the host shared memory,semaphores and message queues inside the container.  Note: the host mode gives the container full access to local shared memory and is therefore considered insecure.
 
 **--link**=[]
-   Add link to another container in the form of name:alias
+   Add link to another container in the form of <name|id>:alias
```

Comment:
Something I noted; because no back-ticks are used in the `.md` files for man, GitHub/MarkDown will treat the `<foo|bar>` notation as HTML tags, making them *invisible* when reading them on GitHub.

This is not something this PR introduced (other parts already use it), but may be something that should be fixed?

I created a screenshot of this section:
![image](https://cloud.githubusercontent.com/assets/1804568/5788846/ac57c040-9e4e-11e4-9cf1-a5eafce71e22.jpg)